README.md

```diff
@@ -59,54 +58,105 @@ Some components or modules in forte may require some [extra requirements](https:
+Forte not only makes it easy to integrate with arbitrary 3rd party tools (Check out these [examples](https://github.com/asyml/forte/tree/master/examples)!), but also brings technology to you by offering a miscellaneous collection of deep learning modules via Texar, and a convenient model-data interface for casting tasks to models.
 
-A simple code example that runs Named Entity Recognizer from Spacy (required
-installing forte spacy wrapper)
+## Quick Start Guide
+Here we provide an example to user to get start with Forte easier. Consider a case we want to get words in the sentence and also extract entities from the sentence. We can write two processors for the two separate tasks.
 
-```python
+First, we imports all required libraries.
+```
 from forte import Pipeline
+from forte.processors.base import PackProcessor
+from forte.data.data_pack import DataPack
+from dataclasses import dataclass
+from forte.data.ontology.top import Generics
+from typing import Optional, Dict, Any
 from forte.data.readers import TerminalReader
 from fortex.spacy import SpacyProcessor
+import re
+```
+In Forte, we use data entry with different entry types to represent data.
+The following class is an example of data entry storing `value`. New User
+doesn't need to pay too much attention about it and can just use it as it is.
 
-for pack in Pipeline().set_reader(
-        TerminalReader()
-).add(
-    SpacyProcessor(), {"processors": ["sentence", "ner"]}
-).initialize().process_dataset():
+```
+@dataclass
+class Word(Generics):
+    """A dummy generic type for words."""
+    value: Optional[str] = None
+    def __init__(self, pack, value):
+        super().__init__(pack)
+        self.value = value
+```
+Next we can write a simple customized processor for the first task, split sentences into words.
+There are two steps for the sentence processing.
+First, we need to strip all punctuation as splitted words should not contain them.
+Second, we need to split stripped sentences into words and write words into the data pack.
+```
+class WordSplitPackProcessor(PackProcessor):
+    """A processor that removes punctuation in sentences in the data pack
+    and split sentences into words and write words into the data pack.
+    """
+    def __init__(self):
+        super().__init__()
+
+    def initialize(self, resources, configs):
+        super().initialize(resources, configs)
+
+    def _process(self, input_pack: DataPack):
+        # write a data type Word into the data pack
+        for sentence in pack.get("ft.onto.base_ontology.Sentence"):
+            # first step: strip all punctuation
+            words = re.sub(r'[^\w\s]','',sentence.text).split(" ")
+            # second step: split stripped sentences into words and write
+            # words into the data pack
+            for w in words:
+                Word(pack=input_pack, value=w)
+
+    @classmethod
+    def default_configs(cls) -> Dict[str, Any]:
+        return {"test": "test, successor"}
+```
```

Comment:
Maybe explain a little bit what is done here, like "get", or how "Word"s are created and stored.

Reply:
How about just using a class from base_ontology instead of creating a new `Word` class here? I don't think people are ready to understand these classes now.